### PR TITLE
larvapoints includes revivable humans

### PR DESCRIPTION
**Pull request by psykzz**
## About The Pull Request

When calculating the number of humans, revivable humans are included

## Why It's Good For The Game

Improves balance of humans / xenos

### CODE REVIEW

**Comment by psykzz** [comment]
Ignore the last commits, wrong PR.

**Comment by terranaut1** [comment]
i like this idea but this implementation of it causes a runtime with get_total_joblarvaworth which ultimately locks the game down since it stops creating burrowed

**Comment by psykzz** [comment]
@terranaut1 it was including dummy humans, which i've now removed.

File: code/modules/mob/living/carbon/human/dummy.dm [hunk 1]
```diff
@@ -5,6 +5,9 @@
 
 INITIALIZE_IMMEDIATE(/mob/living/carbon/human/dummy)
 
+/mob/living/carbon/human/dummy/Initialize()
+	return // This stops dummies being setup and registered in the human_mob_list
```
Comment (Rohesie) [inline code/modules/mob/living/carbon/human/dummy.dm]:
```suggestion
	return INITIALIZE_HINT_NORMAL // This stops dummies being setup and registered in the human_mob_list
```

**Pull request merged**